ai-platform-unified/notebooks/official/custom-training-batch-prediction.ipynb

```diff
@@ -0,0 +1,1213 @@
+{
```

Comment:
<p>"When the batch prediction is done processing, you can finally <strong>you</strong> the predictions"</p><p> </p><p>Is that supposed to be "view" the predictions?</p><p> </p><p>Also this sentence is confusing - could we break it into two shorter sentences to make it easier to parse?</p><p> </p><p>"The predictions will be in a JSONL format, which you indicated at the time you made the batch prediction job, under a subfolder starting with the name <code>prediction</code>, and under that folder will be a file called <code>prediction-results-xxxx-of-xxxx</code>. "</p><p> </p><p>"The predictions will be in a JSONL format, which you indicated when you created the batch prediction job. The predictions are located in a subdirectory starting with the name <code>prediction</code>. Within that directory, there is a file named <code>prediction-results-xxxx-of-xxxx</code>. "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>Done.</p>

Reply:
<p>updated reuse repo</p>